astrobee/survey_manager/survey_planner/tools/survey_planner/command_astrobee

```diff
@@ -0,0 +1,512 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2021, United States Government, as represented by the
+# Administrator of the National Aeronautics and Space Administration.
+# 
+# All rights reserved.
+# 
+# The "ISAAC - Integrated System for Autonomous and Adaptive Caretaking
+# platform" software is licensed under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import argparse
+import os
+import pathlib
+import select
+import socket
+import subprocess
+import sys
+import threading
+from typing import Any, Dict, List
+
+import rospkg
+import rospy
+import yaml
+from ff_msgs.msg import (
+    AckCompletedStatus,
+    AckStamped,
+    CommandArg,
+    CommandConstants,
+    CommandStamped,
+)
+from std_msgs.msg import Header, String
+
+# # Get the directory of the current script
+from survey_planner.problem_generator import load_yaml
+
+# Constants
+MAX_COUNTER = 10
+CHUNK_SIZE = 1024
+
+def get_stereo_traj(config_static, from_bay, to_bay):
+    # Get stereo trajectory
+    traj_matches = [
+        traj
+        for traj in config_static["stereo"].values()
+        if traj["base_location"] == from_bay and traj["bound_location"] == to_bay
+    ]
+    assert (
+        len(traj_matches) == 1
+    ), f"Expected exactly 1 matching stereo trajectory with base {base} and bound {bound}, got {len(traj_matches)}"
+    fplan = traj_matches[0]["fplan"]
+    return fplan
```

Comment:
fixed. However there's the caveat that `plan_interpreter.py` doesn't work on the robot because it imports `matplotlib`. So I change the function location. Feel free to open an issue to organize the code better, for now at least it works on the robot.